helm/cas-ggircs/Chart.yaml

```diff
@@ -8,3 +8,16 @@ dependencies:
   - name: cas-postgres
     version: "0.8.0"
     repository: https://bcgov.github.io/cas-postgres/
+  - name: cas-airflow-dag-trigger
+    version: 1.0.6
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-ggircs-dags
+  - name: cas-airflow-dag-trigger
+    version: 1.0.6
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-swrs-dags
+    condition: download-swrs-dags.enabled
+  - name: cas-airflow-dag-trigger
+    version: 1.0.6
+    repository: https://bcgov.github.io/cas-airflow
+    alias: trigger-ggircs-deploy-db
```

Comment:
I think you have to run a `helm dep up` to update the `Chart.lock` file

Reply:
👍 done!